SU-SOE/stanford_soe_helper
# 7.x 2.x collections soe 2715

# READY FOR REVIEW

# Summary
- This creates a new content type "collection" and sets up and defines the fields used by this content type. 
- A view for the collection page and various blocks are created
- A collection page context is setup
- A new "Manage Content" option is created for Collection in the administration section.


# Needed By (03/28/2018)
- Sprint end

# Criticality
- Client is eagerly awaiting this.

# Steps to Test
- Switch to this branch: https://github.com/SU-SOE/stanford_soe_helper/tree/7.x-2.x-collections-SOE-2715
- Enable the modules: stanford_soe_helper_collection, stanford_soe_helper_collection_administration, stanford_soe_helper_collection_layouts, and stanford_soe_helper_collection_views
- May need to revert feature
- This creates a new content type "collection" and sets up and defines the fields used by this content type. 
- This also adds a entity reference field called "Collection" to the "Magazine article" content type.
- Go to the : node/add/stanford-collection
- Create a new collection
- Navigate to the magazine/collections page to see the new collection list and click on the collection to view the collection page.
- Once the collection is complete, you can edit any magazine article entity and will be able to select the collection to add the article to.
- You should now see the article you included show up on the new collection page
- A new "Manage Content" option is created for Collection in the administration section here: admin/manage/collection.

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2715

## More Information

## Folks to notify
@minorwm, @cjwest, @sherakama 


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
